lib/writer.js

```diff
@@ -346,7 +346,9 @@ module.exports = class Writer extends ReadyResource {
   async _loadNextNode () {
     const seq = this.nodes.length
     if (!(await this.core.has(seq))) return false
-    const { node, checkpoint, maxSupportedVersion } = await this.core.get(seq, { wait: false })
+    const content = await this.core.get(seq, { wait: false })
```

Comment:
We can probably remove the line above and rely only on the null check, but left it in for now because I'm not 100% sure that's its only purpose